### ISSUE TITLE
Got an error while downloading layer from Workbench dataset

### ISSUE THREAD

**Issue by dickens88** [mitre-attack/attack-navigator#405]
Hi, 

I deployed a Navigator v4.5.5 in my local env and integrated with Workbench according to [this guide](https://github.com/center-for-threat-informed-defense/attack-workbench-frontend/blob/master/docs/integrations.md)

```
{
    "versions": [
        {
            "name": "ATT&CK Workbench",
            "domains": [
                {
                    "name": "Enterprise",
                    "data": ["(Enterprise URL)"]
                },
                {
                    "name": "Mobile",
                    "data": ["(Mobile URL)"]
                },
                {
                    "name": "ICS",
                    "data": ["(ICS URL)"]
                }
            ]
        }
    ]
}
```

I created a new layer which is refered to my Workbench instance. after i clicked `download layer as json` then i got an error and nothing downloaded.  

![image](https://user-images.githubusercontent.com/8390209/152668914-10a7fd69-b505-4138-a9ef-56a5f07f82de.png)

I tried to change the `versions.name` in config.json from name `ATT&CK Workbench` to `ATT&CK v10`, then everything works fine. So i suppose there maybe something wrong with the name processing or the integration with Workbench.

The similar error also occurred while uploading json layer file to navigator.

**Comment by clemiller** [comment]
Hi @dickens88,

The error you're receiving stems from an issue in how data versions are handled in the Navigator. The current implementation of the versioning scheme is very rigid, which makes it difficult to configure and upload/download custom data. This is a known bug that we are tracking and should be resolved by the changes proposed in [this issue](https://github.com/mitre-attack/attack-navigator/issues/370).

**Comment by clemiller** [comment]
The changes for https://github.com/mitre-attack/attack-navigator/issues/370 were included in the Navigator v4.6.0 release and appear to have addressed this issue. Closing.
